jupyterlab/labbox_ephys_widgets_jp/src/pluginWidgets.tsx

```diff
@@ -175,6 +175,64 @@ export class SortingViewModel extends DOMWidgetModel {
   static view_module_version = MODULE_VERSION;
 }
 
+interface PluginComponentWrapperProps {
+  plugin: SortingViewPlugin
+  hither: HitherContext
+  sortingObject: any
+  recordingObject: any
+  sortingInfo: any
+  recordingInfo: any
+}
+
+const PluginComponentWrapper: FunctionComponent<PluginComponentWrapperProps> = ({plugin, hither, sortingObject, recordingObject, sortingInfo, recordingInfo}) => {
+  let sorting: Sorting
+  let recording: Recording
+  if (sortingObject.sorting_format) {
+    sorting = {
+      sortingId: '',
+      sortingLabel: '',
+      sortingPath: '',
+      sortingObject,
+      recordingId: '',
+      recordingPath: '',
+      recordingObject,
+      sortingInfo
+    }
+    recording = {
+      recordingId: '',
+      recordingLabel: '',
+      recordingObject,
+      recordingPath: '',
+      recordingInfo
+    }
+  }
+  else {
+    const example = exampleSorting()
+    sorting = example.sorting
+    recording = example.recording
+  }
+
+  const [curation, curationDispatch] = useReducer(sortingCurationReducer, defaultSortingCuration)
```

Comment:
For now we have the curation state in this widget. But obviously we are going to need to hook into a more persistent state so that the curation can be saved somehow.